components/ws-manager/pkg/manager/manager.go

```diff
@@ -109,14 +109,25 @@ const (
 
 // New creates a new workspace manager
 func New(config config.Configuration, client client.Client, rawClient kubernetes.Interface, cp *layer.Provider) (*Manager, error) {
-	wsdaemonConnfactory, _ := newWssyncConnectionFactory(config)
+	wsdaemonConnfactory, err := newWssyncConnectionFactory(config)
+	if err != nil {
+		return nil, err
+	}
+
+	wsdaemonPool := grpcpool.New(wsdaemonConnfactory, checkWSDaemonEntpoint(config.Namespace, client))
+	go func() {
```

Comment:
As a matter of taste, I'd have pulled the Go func and ticker into `grpcpool.New`. Makes the pool easier to use.

Reply:
done